todo_app/data/trello_items.py

```diff
@@ -0,0 +1,184 @@
+from flask import session
+import os
+import json
+import requests
+
+# There is currently an issue with https certificate verification, switch to True once fixed
+_HTTPS_VERIFY_CERTIFICATES = False
+
+class Item:
+    def __init__(self, id, name, status = 'To Do'):
+        self.id = id
+        self.name = name
+        self.status = status
+
+    @classmethod
+    def from_trello_card(cls, card, list):
+        return cls(card['id'], card['name'], list['name'])
+
+url = "https://api.trello.com/1/boards/" + str(os.environ.get('BOARD_ID'))
```

Comment:
Not a problem here but you might find this leads to confusing behaviour in your tests because this line will execute when the file is imported, which means that if your tests are loading their own fake environment variables they might not be set in time for this line to be affected. Easiest solution is probably to just copy this into both the functions that currently reference it